daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Speedy.scala

```diff
@@ -566,6 +568,9 @@ object Speedy {
   }
 
   /** Internal exception thrown when a continuation result needs to be returned. */
-  final case class SpeedyHungry(result: SResult) extends RuntimeException(result.toString)
+  final case class SpeedyHungry(result: SResult) extends RuntimeException {
```

Comment:
Just delaying `result.toString` call to the point when it's needed. Yourkit showed big performance setback here

Reply:
Good catch. 

Though, we probably do not need the message all, since this exception is not an error and is never printed. We could also extends the NoStackTrace to avoid building the stack trace every time

``final case class SpeedyHungry(result: SResult) extends RuntimeException with NoStackTrace``

What do you think @dajmaki ?

Reply:
```suggestion
  final case class SpeedyHungry(result: SResult) extends RuntimeException with NoStackTrace
```

Reply:
`NoStackTrace` sounds like a good idea.